policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_payment(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance payment amount"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_18"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
```

Comment:
Then remove from formula
```suggestion
    defined_for = "qc_fa_eligibility"
```